devito/builtins.py

```diff
@@ -0,0 +1,193 @@
+"""
+Built-in :class:`Operator`s provided by Devito.
+"""
+
+from sympy import Abs, Pow
+import numpy as np
+
+import devito as dv
+
+__all__ = ['assign', 'smooth', 'norm', 'sumall', 'inner']
+
+
+def assign(f, v=0):
+    """
+    Assign a value to a :class:`Function`.
+
+    Parameters
+    ----------
+    f : Function
+        The left-hand side of the assignment.
+    v : scalar, optional
+        The right-hand side of the assignment.
+    """
+    dv.Operator(dv.Eq(f, v), name='assign')()
```

Comment:
We may want to generate a standard library for something that simple at some point to avoid the jit part in some `lib` directory for example.
For future though probably don't need in this PR.

Reply:
not sure how doable that is, as the Operator C code really depends on ``f``